tests/manage/pv_services/test_create_pvc_with_worker_node_down.py

```diff
@@ -0,0 +1,98 @@
+import logging
+import random
+import pytest
+
+from ocs_ci.ocs.resources import pod
+
+from ocs_ci.ocs import node
+from tests import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import ManageTest, tier4, tier4a
+from tests.helpers import get_worker_nodes
+from tests.sanity_helpers import Sanity
+
+log = logging.getLogger(__name__)
+
+
+@tier4
+@tier4a
+class TestCreatePvcWithWorkerNodeDown(ManageTest):
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request, nodes):
+        """
+        Make sure all nodes are up again
+
+        """
+        def finalizer():
+            nodes.restart_nodes_teardown()
+
+        request.addfinalizer(finalizer)
+
+    @pytest.fixture(autouse=True)
+    def init_sanity(self):
+        """
+        Initialize Sanity instance
+        """
+        self.sanity_helpers = Sanity()
+
+    @pytest.mark.polarion_id("OCS-1628")
+    def test_create_delete_pvc_parallel(self, project_factory, storageclass_factory, dc_pod_factory, nodes):
+        """
+        PV provisioning with one worker down
+
+        """
+
+        # Get worker nodes
+        worker_node_list = get_worker_nodes()
+        log.info(f"Current available worker nodes are {worker_node_list}")
+
+        mgr_pod_obj = pod.get_mgr_pods()
+        log.info(mgr_pod_obj[0].name)
+        mgr_node_name = pod.get_pod_node(mgr_pod_obj[0]).name
+        worker_node_list.remove(mgr_node_name)
```

Comment:
Do you think we should try the same test without this safety check?

Reply:
i'm not stopping mgr node is because if mgr node is done we cannot create cephfs pvc and we cannot deleted rbd pvc.

Reply:
I can't remember but it will be migrated to new node?
Single point of failure?

Reply:
yes pod will get migrated to the new node  after 5 min or I can check if mgr node is the one which is stopped node will right one func to deleted the pod so pod will get migrate to a different node

Reply:
Ok thanks

Reply:
Done, modified code if mgr node is selected